pydrive2/test/test_oauth.py

```diff
@@ -11,7 +13,7 @@ def test_01_LocalWebserverAuthWithClientConfigFromFile(self):
     # Delete old credentials file
     self.DeleteOldCredentialsFile('credentials/1.dat')
     # Test if authentication works with config read from file
-    ga = GoogleAuth('settings/test1.yaml')
+    ga = GoogleAuth('pydrive2/test/settings/test1.yaml')
```

Comment:
does it still test what it should be testing? (since we changed to p12)

Reply:
thank you for finding this, this is fixed by renaming `test1.yaml` to `test_oauth_test_01` and reverting to the old test1.yaml content.